src/constructors/keyframes.js

```diff
@@ -6,14 +6,23 @@ import type { Interpolation, NameGenerator } from '../types'
 
 const replaceWhitespace = (str: string): string => str.replace(/\s|\\n/g, '')
 
+let inserted = {}
+
+export const clearKeyframeCache = () => {
```

Comment:
Btw it'd be great if this matched what the `ComponentStyle`s are doing, i.e. the `inserted` cache should be part of the closure and the `test/utils` should recreate the `keyframes` helper to reset the cache.